src/Eloquent/Model.php

```diff
@@ -212,6 +216,8 @@ public function setAttribute($key, $value)
             $value = $builder->convertKey($value);
         }
 
+        $key = (string) $key;
```

Comment:
Cast moved.

> Also, perhaps that comment should be changed since `Builder::convertKey()` also converts strings to Binary UUIDs.

I don't understand what you mean.